docs/index.d.ts

```diff
@@ -270,7 +270,8 @@ export declare interface Scope {
    * @param {Span} [span=scope.active()] The span to activate.
    * @returns The bound target.
    */
-  bind<T>(fn: ((...args: any[]) => T | void), span?: Span | null): ((...args: any[]) => T | void);
+  bind<T extends (...args: any[]) => void>(fn: T, span?: Span | null): T;
```

Comment:
Why is this one necessary in addition to the one below?

Reply:
This was confusing to me as well. If I remove this line, the TS compiler complains and seems to match the function that returns void passed in the `bindFunctionVoidType` example to the bind definition accepting an EventEmitter.

Reply:
It seems to work with the old syntax without `void` as well, but I guess your version preserves the original type of the function. I think `tracer.wrap()` may has the same limitation. If you have some time, it would be awesome to fix it in the same PR, otherwise no worries I'll make another one later :)

Reply:
@rochdev I will defer to you :) have not yet upgraded to 0.10.